tests/std/include/test_header_units_and_modules.hpp

```diff
@@ -471,6 +475,12 @@ void test_mutex() {
 void test_new() {
     using namespace std;
     puts("Testing <new>.");
+
+    static_assert(is_class_v<bad_alloc>);
+    static_assert(is_class_v<bad_array_new_length>);
+    static_assert(is_same_v<underlying_type_t<align_val_t>, size_t>);
+    static_assert(is_class_v<nothrow_t>);

```

Comment:
Should we `(void)nothrow;` here (or something) to esnure `nothrow` works as well?

Reply:
I only see red and green =)